apps/cookbook/src/app/examples/accordion-example/accordion-example.component.html

```diff
@@ -8,3 +8,6 @@ <h2>Accordion with item initially expanded</h2>
 
 <h2>Accordion inside a card</h2>
 <cookbook-accordion-card-example></cookbook-accordion-card-example>
+
+<h2>Accordion with disabled item</h2>
+<cookbook-accordion-disabled-example></cookbook-accordion-disabled-example>
```

Comment:
I actually think we could just make e.g. Accordion Item 3 in the default example disabled instead. Dont think we need a new example for this.